docs/contents/dev-docs/testing.md

```diff
@@ -73,9 +73,58 @@ Finally, if you would like to do functional testing, where you would like to cre
 in your account using `KinD` cluster, please use `-r` and pass in `AWS ROLE ARN` to `scripts/kind-build-test.sh` script or
 `make kind-cluster-functional SERVICE=ecr ROLE_ARN=arn:aws:iam::12345678980:role/Admin-k8s`
 
-Note: For above functional test, `sts-assume-role.sh` under `scripts/lib` directory will 
-make `aws sts assume-role --role-session-arn $AWS_ROLE_ARN --role-session-name $TEMP_ROLE` API call to fetch `AWS_ACCESS_KEY_ID`, 
-`AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` variable and inject them as secrets to the deployment. The duration of the session token is 900 seconds (15 minutes)
+Note: For above functional testing, `generate_temp_creds` function under `scripts/lib/aws.sh` script will 
```

Comment:
Please use the respective [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/), in this case `!!! note`.